examples/metrics-basic/src/main.rs

```diff
@@ -6,7 +6,16 @@ use std::error::Error;
 use std::vec;
 
 fn init_meter_provider() -> opentelemetry_sdk::metrics::SdkMeterProvider {
-    let exporter = opentelemetry_stdout::MetricsExporterBuilder::default().build();
+    let exporter = opentelemetry_stdout::MetricsExporterBuilder::default()
+        // Build exporter using Default (Cumulative) Temporality Selector.
+        .with_temporality_selector(
```

Comment:
@utpilla suggested this can be further simplified. An enum should be better here, delta/cumulative/lowmemory being the enum variants.

@pitoniak32 Do you have bandwidth to help make this change? Hoping to make it done by next release, so as to avoid breaking users after that release.

Reply:
Yeah for sure, I can take a look tomorrow morning. When would the next release be? Also, Is there an open issue for this currently?

Reply:
Next release is ~Oct 22-25.
No issue with all the details, https://github.com/open-telemetry/opentelemetry-rust/issues/2099 is the closest. Hope that is good enough to get started.

Reply:
I took a first go at this before work this morning. I will take another look and try and clean things up later today.